tox.ini

```diff
@@ -12,15 +12,21 @@ deps=
     -roverride_requirements.txt
     cuda101: cupy-cuda101
     cuda102: cupy-cuda102
+    cuda110: cupy-cuda110
 extras=
     hdbscan
 setenv=
     # Using pytest in combination with tox on files that are part of the installed package
     # leads to collisions between the local source tree and the installed package when running tests.
     # See https://github.com/pytest-dev/pytest/issues/2042
     PY_IGNORE_IMPORTMISMATCH=1
-    # Debug asyncio problems:
-    PYTHONASYNCIODEBUG=1
+    # Debug asyncio problems - has some perf overhead, so only enable if there is a problem
+    # PYTHONASYNCIODEBUG=1
+    # Dask configuration to reduce background load:
+    DASK_DISTRIBUTED__ADMIN__TICK__INTERVAL=1000
+    DASK_DISTRIBUTED__SCHEDULER__WORK_STEALING_INTERVAL=1000
+    DASK_DISTRIBUTED__WORKER__PROFILE__CYCLE=60000
+    DASK_DISTRIBUTED__WORKER__PROFILE__INTERVAL=30000
```

Comment:
Ref #61 -- would it make sense to use that by default in LiberTEM?

Reply:
Yeah, we could possibly set a few of these when creating a local cluster. We should try to measure this, to check where the tweaking actually makes sense (there are some stats in `Worker.digests`, need to look at how to access them)